app/dashboard/notifications.py

```diff
@@ -418,33 +418,35 @@ def build_github_notification(bounty, event_name, profile_pairs=None):
     bounty_owner = f"@{bounty.bounty_owner_github_username}" if bounty.bounty_owner_github_username else ""
     status_header = get_status_header(bounty)
 
+    crowdfund_msg = f"* Want to chip in? Add your own contribution [here]({absolute_url})."
+    openwork_msg = f"* ${amount_open_work} more funded OSS Work available on the " \
+                   f"[Gitcoin Issue Explorer](https://gitcoin.co/explorer)"
```

Comment:
You can drop `f` here and throughout all of the strings below that don't have a `{}` present on the unique line.

Reply:
i think its easier from a readability (and for consistency and preventing stray dynamic variables in non-f-strings) standpoint to have them all f strings, and the performance impact is negligible.